### PR TITLE
Create production and daily development egg

### PR DESCRIPTION
**Pull request by sbrunner**
Required to integrate c2cgeoform into c2cgeoportal build tool chain.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,3 +31,26 @@ before_script:
 - psql -d c2cgeoform_demo_tests -c "CREATE EXTENSION postgis;" -U postgres
 
 script: make check && make test
+
+before_deploy:
+- make compile-catalog
```
Comment (arnaud-morvan) [inline .travis.yml]: Not sure this is required as it is done by make build, and translations should be tested.

File: .travis.yml [hunk 2]
```diff
@@ -27,3 +31,26 @@ before_script:
...
+  server: https://upload.pypi.org/legacy/
+  user: sbrunner
+  password:
+    secure: "PoIuVu4sdApkWm6CXnBMMzXCj1vNnEdLXKqcBkXlKjBxh6JlivGRmqoFizJAe3KNjZMwMBMcJkMR+f53g0z3P06sIx9WQCaAjYV0j5w5Si71tjjOWohY2DjIaUcNNOsodZc9KDdtSpN0Xm4+vbn7Lg9FofCovs2KAIsBONVD/I4="
+  distributions: sdist bdist_wheel
+  skip_upload_docs: true
+  skip_cleanup: true
+  on:
+    tags: true
+    repo: camptocamp/c2cgoform
```
Comment (arnaud-morvan) [inline .travis.yml]: camptocamp/c2cg **e** oform

File: setup.py [hunk 3]
```diff
@@ -14,31 +14,35 @@
...
-      entry_points="""\
-      [paste.app_factory]
-        main=c2cgeoform:main
-      [pyramid.scaffold]
-        c2cgeoform=c2cgeoform.scaffolds:C2cgeoformTemplate
-      """,
-      )
+setup(
+    name='c2cgeoform',
+    version='0.0',
```
Comment (arnaud-morvan) [inline setup.py]: Could we use version=2.0 to be consistent with git branches

File: .travis.yml (hunk 1 above)
Comment (sbrunner) [reply .travis.yml]: But Travis clean the build result before running the deploy ...

File: .travis.yml (hunk 2 above)
Comment (sbrunner) [reply .travis.yml]: Oups...

File: setup.py (hunk 3 above)
Comment (sbrunner) [reply setup.py]: +1, thanks :-)

File: .travis.yml (hunk 1 above)
Comment (arnaud-morvan) [reply .travis.yml]: Ok thanks

**Pull request merged**